packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -853,6 +922,11 @@ function convertToBasePropertyTypeUntagged(
 	}
 }
 
+/**
+ * TODO
```

Comment:
Syntactic sugar helper function for tagging all given properties with the same tag.
Suggested usage may be something like this, where this line is amidst other properties being added to a telemetry event:

...tagData("someTag", { foo: 1, bar: 2 }),

This will result in foo and bar added to the event with their values tagged.